internal/driver/constants.go

```diff
@@ -29,9 +29,9 @@ const (
 	FrameRateValueDenominator       = "FrameRateValueDenominator"
 	FrameRateValueNumerator         = "FrameRateValueNumerator"
 	PathIndex                       = "PathIndex"
-	PixFmtWidth                     = "Width"
-	PixFmtHeight                    = "Height"
-	PixFmtPixFmt                    = "PixelFormat"
+	Width                           = "Width"
+	Height                          = "Height"
+	PixFormat                       = "PixelFormat"
```

Comment:
I think I would recommend just removing the PixFormat const and use the existing PixelFormat one as otherwise the name and values do not match.